src/codeflare_sdk/cluster/cluster.py

```diff
@@ -156,11 +161,42 @@ def up(self):
                         plural="appwrappers",
                         body=aw,
                     )
+                print(f"AppWrapper: '{self.config.name}' has successfully been created")
             else:
                 self._component_resources_up(namespace, api_instance)
+                print(
+                    f"Ray Cluster: '{self.config.name}' has successfully been created"
+                )
         except Exception as e:  # pragma: no cover
             return _kube_api_error_handling(e)
 
+    def _cluster_up_down_buttons(self):
+        delete_button = widgets.Button(
+            description="Cluster Down",
+            icon="trash",
+        )
+        up_button = widgets.Button(
+            description="Cluster Up",
+            icon="play",
```

Comment:
You could add a `tooltip` and include a small description on what the button does. The user will be able to see the message when hovering over the button.
Reference: https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20List.html#togglebutton

Reply:
That's pretty cool. I'll add descriptions